app/src/main/java/vn/realtest/stock/justanewproject/Fragments/FavoriteFragment.java

```diff
@@ -41,25 +46,30 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         view = inflater.inflate(R.layout.fragment_favorite, container, false);
         marketStockList = new ArrayList<>();
         rv_favorite = (RecyclerView) view.findViewById(R.id.rv_favorite);
-        mAdapter = new MarketAdapter(marketStockList);
+        mAdapter = new HoseAdapter(marketStockList);
         RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getContext());
         rv_favorite.setLayoutManager(mLayoutManager);
         rv_favorite.setAdapter(mAdapter);
         prepareStockData();
+        receiveData();
         return view;
     }
 
-    private void prepareStockData() {
-
-        MarketStock marketStock = new MarketStock("GMD", "31.5", "-1.36%", "Vol: 3000");
-        marketStockList.add(marketStock);
+    private void receiveData() {
+        MainActivity activity = (MainActivity) getActivity();
+        stock_name_data = activity.getStockName();
+        stock_index_data = activity.getStockIndex();
+        id_data = activity.getIdData();
+        if (!(stock_name_data.isEmpty() && stock_index_data.isEmpty())) {
+            Toast.makeText(view.getContext(), "Trade: stock name: " + stock_name_data + " stock index: " + stock_index_data + " id_san: " + id_data, Toast.LENGTH_SHORT).show();
```

Comment:
I recommend using Log.d instead of Toast because of the latency of Toast then we cannot follow all events